policyengine_us/variables/gov/states/ms/tax/income/adjustments/ms_retirement_income_exemption.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_us.model_api import *
+
+
+class ms_retirement_income_exemption(Variable):
+    value_type = float
+    entity = Person
+    label = "Mississippi retirement income exemption"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://law.justia.com/codes/mississippi/title-27/chapter-7/article-1/section-27-7-15/"  # (4)(k)
+    defined_for = StateCode.MS
+
+    def formula(person, period, parameters):
+        excluded_income = add(
+            person,
+            period,
+            [
+                "social_security_retirement",
```

Comment:
If MS AGI starts with federal AGI
```suggestion
                "taxable_social_security",
```

Reply:
yep, thanks